docs/en/week02/02-1.md

```diff
@@ -71,11 +71,15 @@ Loss function is a function that is minimized during training. There are two typ
 $$
  L(x,y,w) = C(y, G(x,w))
 $$
+
 2) Average Loss -
 
-​	For any set of Samples $$S = \{(x[p],y[p]) \mid p=0,1...P-1 \}$$
+​	For any set of Samples 
+$$
+  S = \{(x[p],y[p]) \mid p=0,1...P-1 \}
+$$
 
-​	Average Loss over the Set S is given by :  $$L(S,w) = \frac{1}{P} \sum_{(x,y)} L(x,y,w)$$
+​	Average Loss over the Set $S$ is given by :  $$L(S,w) = \frac{1}{P} \sum_{(x,y)} L(x,y,w)$$
```

Comment:
Like this kind of edits is needed for the [`ES`](https://github.com/Atcold/pytorch-Deep-Learning/blame/b8231a30636b9552ae1980a9be6f9c3de967eb14/docs/es/week02/02-1.md#L146), and [`ZH`](https://github.com/Atcold/pytorch-Deep-Learning/blame/b8231a30636b9552ae1980a9be6f9c3de967eb14/docs/zh/week02/02-1.md#L113) versions.